Fix existing CheckStyle issues (B..BlockLava)

Pr0methean opened the pull request.

Pr0methean committed:
Merge remote-tracking branch 'refs/remotes/GlowstoneMC/dev' into style1

Pr0methean committed:
Fix existing CheckStyle issues (A)

First of several commits. Classes are being processed alphabetically by
simple name.

Pr0methean committed:
Merge remote-tracking branch 'refs/remotes/GlowstoneMC/dev' into style1

Pr0methean committed:
Fix existing Checkstyle issues (B-BlockLava)

Pr0methean committed:
Fix existing Checkstyle issues (B-BlockLava) -- second pass

mastercoms commented on src/main/java/net/glowstone/block/blocktype/BlockCrops.java:
```diff
@@ -68,12 +68,12 @@ public boolean canTickRandomly() {
         int cropState = block.getData();
-        // we check light level on the above block, meaning crops needs at least one free block above it
-        // in order to grow naturally (vanilla behavior)
+        // we check light level on the above block, meaning crop needs at least one free block above
```
I think it should read "the crops need" and then "them" instead of it

Pr0methean replied:
Done.

mastercoms commented on src/main/java/net/glowstone/block/blocktype/BlockDirectDrops.java:
```diff
@@ -16,29 +16,63 @@
+     * @param amount the amount to drop
+     * @param neededTool the tool(s) that can break this block
+     */
     public BlockDirectDrops(Material dropType, int data, int amount, MaterialMatcher neededTool) {
```
I think the description here should be "Creates a block type that drops an item directly when broken", or something along those lines. Adapt the others to say something similar as well.

Pr0methean replied:
Done.

Pr0methean committed:
Revise BlockDirectDrops Javadoc summaries

Per 632#discussion_r158409799

Pr0methean committed:
Comment edit

Per
https://github.com/GlowstoneMC/Glowstone/pull/632#pullrequestreview-85219068
